_posts/2020/11/2020-11-23-python-repository.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+date: 2020-11-23
+title: "Some name"
+author: olenagerasimova
+description: This blog post shows how binary and pypi repositories can be used.
+keywords:
+  - artipie
+  - private
+  - pypi
+  - python
+  - repository
+  - registry
+  - binary
+---
+
+Being a student I dreamed about something like Artipie, I needed some private centralized storage 
+where I could have kept both binary or text data files and Python packages. We were conducting a research
+and used to work with a lot of data from experimental measurements and used Python to process and 
+visualise them. My university colleagues from that time were mathematicians, they do not have strong
```

Comment:
@olenagerasimova it seems that "they do not have strong..." should be "they did not have strong..." as the statement is in past tense